go.mod

```
@@ -1,6 +1,6 @@
 module github.com/enterprise-contract/enterprise-contract-controller
 
-go 1.20
+go 1.20.10
```

Let's go with a newer version (and 1.20 doesn't do x.y.z versions):
```suggestion
go 1.21.4
```